scripts/workload_analyzation.py

```
@@ -30,6 +51,8 @@ def get_code_level(filename):
         return CodeLevel.BUILDER
     elif 'common' in file_dir:
         return CodeLevel.COMMON
+    elif 'scripts' or 'CMakeLists.txt' in file_dir:
```

Ruff (0.8.2)

Use `True` instead of `True or ...`

Replace with `True`

(SIM222)